src/popup/pages/SignTransaction/index.js

```diff
@@ -233,12 +231,30 @@ class SignTransaction extends React.Component {
       </div>
     )
   }
+  chainLabel = (chainContext) => {
+    switch (chainContext) {
+      case '53852332637bacb61b91b6411ab4095168ba02a50be4c3f82448438826f23898':
+        return 'Oasis Protocol Foundation Mainnet - Cobalt'
+      case '5ba68bc5e01e06f755c4c044dd11ec508e4c17f1faf40c0e67874388437a9e55':
+        return 'Oasis Protocol Foundation Testnet - 2021-04-13'
```

Comment:
config.js or similar is a better place to put these

Reply:
oh thanks I'll move them

Reply:
moved